.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+exclude: ^docs/
+
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
+  hooks:
+  - id: check-added-large-files
```

Comment:
any specific reason for checking for large files? 🤔 I don't think thats required.
Also, adding `requirements-txt-fixer` and `check-yaml` might be helpful.

Reply:
This is a template that was taken from another project that I worked on to check the feasibility of including it in this project, you can make any modifications that you see fit.

Reply:
okay 👍